### ISSUE TITLE
[Feature Request]: Apply graphics settings changes directly, without closing the graphics menu

### ISSUE THREAD

**Issue by Firedavinci** [PCSX2/pcsx2#5921]
### Description

Currently the PCSX2 graphics settings changes only apply once you hit the "ok" button which will close the graphic settings menu entirely.

This is annoying if you want to compare different settings with each other (on vs off), or you want to see the effect of a certain setting for example. This way the graphic settings menu has to be re-opened with every graphic setting change, since it closes everytime you hit the "OK" button.

Suggestion is to make settings apply immediately. Once the graphic setting box is ticked or the setting has been chosen it should apply directly, without closing the graphics menu.

This way we can see in real time the changes and make comparisons.

**Note:** [Dolphin emulators graphic settings has such a feature for example]

### Reason

It will make graphic setting changes easier and is a quality of life feature (QoL) improvement.
Graphic setting changes can be seen in real time this way, without having to re-open the graphic settings menu every time.

### Examples

[Dolphin emulator's graphic settings has such a feature for example] 
Graphic settings apply in real time, without the need of apply (ok) button and without closing the graphic settings menu.

**Comment by refractionpcsx2** [comment]
This already happens in the new Qt interface, and there's little reason to do it in the current WX one since we're dropping it soon.

So, already done, thanks :)

**Comment by Firedavinci** [comment]
Is it public yet?

**Comment by refractionpcsx2** [comment]
Not yet, no, we've still got some work to do before we can.

**Comment by Firedavinci** [comment]
Goot to know. So its comming then? :)

**Comment by refractionpcsx2** [comment]
yes, it is coming, it's mostly done :)

**Comment by Firedavinci** [comment]
Will the new interface be like lets say dolphin emulator, where games can be chosen by clicking on game-image-tiles?

**Comment by refractionpcsx2** [comment]
yes, and it will have per game configs and alsorts of other nice things.

It's essentially Duckstation's interface (which is also similar to Dolphin's)

**Comment by Firedavinci** [comment]
Oh, thats nice :) Finally lol. So what about the Retroarch pcsx2 core? Will this be dropped?

**Comment by refractionpcsx2** [comment]
It was never picked up, we have nothing to do with it, never did.

**Comment by Firedavinci** [comment]
Oh really? Whos working on that then? Always thought the cores are made by the guys who made the main emu as well.

**Comment by refractionpcsx2** [comment]
No clue, it's 3rd party stuff, I think RA are maintaining it themselves, idk.

**Comment by Firedavinci** [comment]
Ok, I see. Hey thanks for the chat. God speed to you and the new interface :)
